charts/pulsar/values.yaml

```diff
@@ -776,6 +776,11 @@ broker:
 ##
 functions:
   component: functions-worker
+  ## Pulsar: Functions Worker ClusterRole or Role
+  ## templates/broker-rbac.yaml
+  # Default is false which deploys functions with ClusterRole and ClusterRoleBinding at the cluster level
+  # Set to true to deploy functions with Role and RoleBinding inside the specified namespace
+  limit_to_namespace: false
```

Comment:
@bsheltonihs can you make this setting to be `functions.rbac.limit_to_namespace`? it is probably better to be scoped under `functions.rbac` so people understand that it is a `rbac` related setting.

Reply:
took the suggestion/recommendation and moved it under `functions.rbac.limit_to_namespace`